src/main/scala/com/cognite/sdk/scala/v1/resources/sessions.scala

```diff
@@ -51,6 +51,14 @@ class Sessions[F[_]](val requestSession: RequestSession[F])
         uri"$baseUrl/token",
         value => value
       )
+
+  def revoke(ids: Items[CogniteInternalId]): F[Seq[CogniteInternalId]] =
```

Comment:
later on we may want to add other response fields, but as a start ids could be enough